ledger/src/blockstore_db.rs

```diff
@@ -141,7 +141,15 @@ pub mod columns {
     pub struct SlotMeta;
 
     #[derive(Debug)]
-    /// The orphans column
+    /// The orphans column.
+    ///
+    /// This column family tracks whether a slot has a parent.  If a slot does
+    /// not have a parent yet, it is by definition an orphan slot and will have
+    /// an entry in this column family with true value indicating it is an
+    /// orphan.
```

Comment:
I think this is good but is slightly repetitive and could be slimmed down; maybe:

> This column family tracks whether a slot has parent; slots without a parent are by definition orphan slots. Orphans will have an entry in this column family with true value. If a slot's parent is inserted, the entry will be deleted.